webapp/tests/test_functions.py

```diff
@@ -3457,7 +3686,6 @@ def mock_data_fetcher(reqCtx, path_expression):
 
 
         with patch('graphite.render.evaluator.fetchData', mock_data_fetcher):
-            # input values will be ignored and replaced by regression function
```

Comment:
Why removing this and comments below? Just curious

Reply:
Same question, looks great otherwise

Reply:
They were erroneously copied from other checks when I created them this week.  I'll see if I can put comments there that do make sense.